awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml

```diff
@@ -17,12 +17,12 @@
 spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
-    value: public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8
+    value: public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-cpu-py39-ubuntu20.04-e3-v1.2
     # The list of available standard container Images
     options:
-    - kubeflownotebookswg/jupyter-scipy:v1.6.0
-    - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-gpu-py38-cu112-ubuntu20.04-v1.8
-    - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8
+    - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.6.0
+    - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-gpu-py39-cu112-ubuntu20.04-e3-v1.2
+    - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-cpu-py39-ubuntu20.04-e3-v1.2
     - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.11.0-gpu-py38-cu115-ubuntu20.04-e3-v1.1
```

Comment:
pytorch has a 1.12 image
https://github.com/aws/deep-learning-containers/blob/master/available_images.md#ec2-framework-containers-tested-on-ec2-ecs-and-eks-only

Reply:
Oh, I checked the Github releases pages for 1.12 but only one of the images and assumed it was not all released yet - DLC github releases and these docs don't seem to be in sync. Anyway, this helps us avoid the tricky tag update for Pytorch so let me make that change.

Reply:
Looks like the tags have changed again, hence my confusion. I am working on this change.